.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Kindly add `node_modules/` folder here as well, even if we don't have it in our files, we should always list it here in case we use npm packages in the future